README.md

```diff
@@ -109,35 +153,131 @@ Now in a slurm job array the work will be performed (you can set the number of a
 
 ## Running on Grid
 
-### Fractal example
+In this fractal example we will implement the following pilot job workflow:
+
+* First we define and generate the application tokens with all the necessary parameters.
+* Then we define and create a shell script to process one task (*process_task.sh*) that will be sent with the job using the input sandbox. This contains some boiler plate code to e.g. setup the environment, download software or data from the Grid storage, run the application etc. This doesn’t have to be a shell script, however, setting up environment variables is easiest when using a shell script, and this way setup scripts are separated from the application code.
+* We also define and create a Python script to handle all the communication with the token pool server, call the process_task,sh script, catch errors and do the reporting.
+* Finally we define the :abbr:`JDL (Job Description Language)` on the User Interface machine to specify some general properties of our jobs. This is required to submit a batch of pilot jobs to the Grid that will in turn initiate the Python script as defined in the previous step.
+
+
+### Prerequisites
+
+To be able to run the example you must have:
+
+* All the three Grid :ref:`prerequisites` (User Interface machine, Grid certificate, VO membership)
+* An account on PiCaS server (send your request to <helpdesk@surfsara.nl>)
+
+
+
+### Picas sample example
+
+
+* Log in to the :abbr:`UI (User Interface)` and download the :download:`pilot_picas_fractals.tgz </Scripts/picas-python3/pilot_picas_fractals.tgz>` example, the couchdb package for Python :download:`couchdb.tgz </Scripts/couchdb.tgz>` and the fractals source code :download:`fractals.c </Scripts/fractals.c>`.
+
+* Untar ``pilot_picas_fractals.tgz`` and inspect the content:
+
+```
+tar -xvf pilot_picas_fractals.tgz
+cd pilot_picas_fractals/
+ls -l
+-rwxrwxr-x 1 homer homer 1247 Jan 28 15:40 createTokens
+-rw-rw-r-- 1 homer homer 1202 Jan 28 15:40 createTokens.py
+-rw-rw-r-- 1 homer homer 2827 Jan 28 15:40 createViews.py
+-rw-rw-r-- 1 homer homer  462 Jan 28 15:40 fractals.jdl
+drwxrwxr-x 2 homer homer  116 Jan 28 15:40 sandbox
+```
+
+Detailed information regarding the operations performed in each of the scripts below is embedded to the comments inside each of the scripts individually.
 
-On the grid, in our screnario, you need to supply the entire environment through the sandbox (a more grid-native CVMFS example is available in the [picas-profile](https://github.com/sara-nl/picas-profile) repository). The binaries and python code need to be in this sandbox.
-First we need to create a tar of the picas code, so that it can be sent to the Grid:
+* Also download the current PiCaS version :download:`picas.tar </Scripts/picas-python3/picas.tar>` and put both PiCaS and the couchdb.tgz file in the ``sandbox`` directory:
 
 ```
-tar cfv grid-sandbox/picas.tar ../picas/
+cd sandbox
+mv ../../couchdb.tgz ./
+mv ../../picas.tar ./
 ```
```

Comment:
Only for the grid part of the fractals example explain how to create a tar of the picas code, so that it can be sent to the Grid